### PR TITLE
move build configuration into `pyproject.toml`

### PR DESCRIPTION
**Pull request by zacharyburnett**
use PEP621 and `ini2toml` / `peppyproject` to move metadata into `pyproject.toml`

### CODE REVIEW

**Comment by WilliamJamieson** [comment]
Can this wait until #112 is merged?

**Review by braingram — APPROVED** [review]
Looks good to me.

**Pull request merged**
